keps/sig-network/20180612-ipv4-ipv6-dual-stack.md

```diff
@@ -15,7 +15,7 @@ approvers:
   - "thockin@"
```

Comment:
after checking this issue https://github.com/kubernetes/kubernetes/pull/86895
I think we should add sig-api-machinery
/cc @liggitt @smarterclayton

Reply:
We are also missing `- [Version Skew Strategy](#version-skew-strategy)`

per https://github.com/kubernetes/kubernetes/issues/86773

Reply:
Thanks for raising this